ui/component/common/nag.jsx

```diff
@@ -10,17 +10,24 @@ type Props = {
   actionText: string,
   href?: string,
   type?: string,
+  inline?: boolean,
   onClick?: () => void,
   onClose?: () => void,
 };
 
 export default function Nag(props: Props) {
-  const { message, actionText, href, onClick, onClose, type } = props;
+  const { message, actionText, href, onClick, onClose, type, inline } = props;
 
   const buttonProps = onClick ? { onClick } : { href };
 
   return (
-    <div className={classnames('nag', { 'nag--helpful': type === 'helpful', 'nag--error': type === 'error' })}>
+    <div
+      className={classnames('nag', {
+        'nag--helpful': type === 'helpful',
+        'nag--error': type === 'error',
+        'nag--inline': inline,
```

Comment:
Add support for inline nags (used on the player). Inline is possibly a bad term as I review this, should possibly be absolute?